mediator/src/main/kotlin/no/nav/dagpenger/rapportering/personregister/mediator/connector/MeldekortregisterConnector.kt

```diff
@@ -0,0 +1,80 @@
+import no.nav.dagpenger.rapportering.personregister.mediator.Configuration
+import no.nav.dagpenger.rapportering.personregister.mediator.metrikker.ActionTimer
+
+class MeldekortregisterConnector(
+    private val meldekortregisterUrl: String = Configuration.meldekortregisterUrl,
+    private val meldekortregisterTokenProvider: () -> String? = Configuration.meldekortregisterTokenProvider,
+    private val httpClient: HttpClient = createHttpClient(),
+    private val actionTimer: ActionTimer,
+) {
+    suspend fun oppdaterIdent(
+        ident: String,
+        nyIdent: String,
+    ) = withContext(Dispatchers.IO) {
+        val result =
+            sendPostRequest(
+                httpClient = httpClient,
+                endpointUrl = "$meldekortregisterUrl/oppdater_ident",
+                token =
+                    meldekortregisterTokenProvider.invoke()
+                        ?: throw RuntimeException("Klarte ikke å hente token"),
+                metrikkNavn = "meldekortregister_oppdaterIdent",
+                body = OppdaterIdentRequest(ident, nyIdent),
+                parameters = mapOf(),
+                actionTimer = actionTimer,
+            ).also {
+                logger.info { "Kall til meldekortregister for å oppdatere ident ga status ${it.status}" }
+            }
+
+        if (result.status != HttpStatusCode.OK) {
+            logger.error { "Uforventet status ${result.status.value} ved oppdatering av ident i meldekortregister" }
+            throw RuntimeException("Uforventet status ${result.status.value} ved oppdatering av ident i meldekortregister")
+        }
+    }
+
+    suspend fun konsoliderIdenter(
+        gjeldendeIdent: String,
+        identer: List<String>,
+    ) = withContext(Dispatchers.IO) {
+        val result =
+            sendPostRequest(
+                httpClient = httpClient,
+                endpointUrl = "$meldekortregisterUrl/konsolider_identer",
+                token =
+                    meldekortregisterTokenProvider.invoke()
+                        ?: throw RuntimeException("Klarte ikke å hente token"),
+                metrikkNavn = "meldekortregister_konsoliderIdenter",
+                body = KonsoliderIdenterRequest(gjeldendeIdent, identer),
+                parameters = mapOf(),
+                actionTimer = actionTimer,
+            ).also {
+                logger.info { "Kall til meldekortregister for å konsolidere identer ga status ${it.status}" }
+            }
+
+        if (result.status != HttpStatusCode.OK) {
+            logger.error { "Uforventet status ${result.status.value} ved konsolidering av identer i meldekortregister" }
+            throw RuntimeException("Uforventet status ${result.status.value} ved konsolidering av identer i meldekortregister")
+        }
+    }
+
+    companion object {
+        private val logger = KotlinLogging.logger {}
+    }
+
+    data class OppdaterIdentRequest(
+        val ident: String,
+        val nyIdent: String,
+    )
```

Comment:
Her burde vi egentlig ha med personId også. Meldekortregisteret er avhengig av å ha samme personId som personregistrert for at oppslag osv skal fungere